src/components/form/dt-multiselect-button-group/dt-multiselect-button-group.js

```diff
@@ -37,18 +50,26 @@ export class DtMultiSelectButtonGroup extends DtFormBase {
 
   constructor() {
     super();
-    this.buttons= [];
+    this.buttons = [];
     this.selectedButtons = [];
     this.value = [];
+    this.custom = true;
   }
 
   connectedCallback() {
     super.connectedCallback();
     this.selectedButtons = this.value ? this.value.map(button => ({ value: button })) : [];
   }
+  
 
-  _handleButtonClick(event) {
+  _handleButtonClick(event, label) {
     const buttonValue = event.target.value;
+    if (buttonValue === "milestone_baptized" && this.isModal && this.isModal.includes(label) && !this.value.includes("milestone_baptized")) {
```

Comment:
I get a console error because this.value is null, if I remove the !this.value.includes("milestone_baptized") check it works. What is this check checking for?